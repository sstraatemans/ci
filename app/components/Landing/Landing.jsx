import React from 'react';
import { Link } from 'react-router';
import { FormattedMessage } from 'react-intl';

import {Row, Panel} from './../Panel';

const Landing = React.createClass({


  render () {
    return (
      <div className='landing'>
        <Row>
          <Panel key="2" showMore="true">test<br/>sdfsdf<br/><br/>sdf<br/>sdsf<br/>sdf<br/>ssdf<br/><br/><br/>sdfsdfssdf<br/><br/><br/>sssdfsdf</Panel>
          <Panel key="1">test<br/>sdfsdf<br/><br/>sdf<br/>sdsf<br/>sdf<br/>ssdf<br/><br/><br/>sdfsdf</Panel>
        </Row>


      </div>
    );
  }
});

export const Unwrapped = Landing;

export default Landing;
