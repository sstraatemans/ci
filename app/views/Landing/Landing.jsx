import React from 'react';
import { Link } from 'react-router';
import { FormattedMessage } from 'react-intl';
import {Row, Panel} from './../../components/Panel';
import Login from './../../components/Login';

class Landing extends React.Component{
  render () {
    return (
      <div className='landing'>
        <Login />
        <Row>
          <Panel key="2" showMore="true">test<br/>sdfsdf<br/><br/>sdf<br/>sdsf<br/>sdf<br/>ssdf<br/><br/><br/>sdfsdfssdf<br/><br/><br/>sssdfsdf</Panel>
          <Panel key="1">test<br/>sdfsdf<br/><br/>sdf<br/>sdsf<br/>sdf<br/>ssdf<br/><br/><br/>sdfsdf</Panel>
        </Row>
      </div>
    );
  }
};

export const UnwrappedLanding = Landing;
export default Landing;
