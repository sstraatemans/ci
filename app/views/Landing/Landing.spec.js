/* eslint-env node, jest */
import React from 'react';
import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Landing, {UnwrappedLanding} from './Landing';


test('Landing snapshot', () => {
  const component = shallow(<UnwrappedLanding />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});

// test('Laning should render word landing', () => {
//   const component = render(<Landing />);
//   expect(component.find('.landing').text()).toEqual('landing');
// });
