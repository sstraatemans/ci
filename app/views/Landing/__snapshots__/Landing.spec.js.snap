exports[`test Landing snapshot 1`] = `
<div
  className="landing">
  <Row>
    <Panel
      showMore="true">
      test
      <br />
      sdfsdf
      <br />
      <br />
      sdf
      <br />
      sdsf
      <br />
      sdf
      <br />
      ssdf
      <br />
      <br />
      <br />
      sdfsdfssdf
      <br />
      <br />
      <br />
      sssdfsdf
    </Panel>
    <Panel>
      test
      <br />
      sdfsdf
      <br />
      <br />
      sdf
      <br />
      sdsf
      <br />
      sdf
      <br />
      ssdf
      <br />
      <br />
      <br />
      sdfsdf
    </Panel>
  </Row>
</div>
`;
